src/Internal/Payment/PaymentMethod/NewPaymentMethod.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Class NewPaymentMethod
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal\Payment\PaymentMethod;
+
+/**
+ * Representation of a newly entered payment method.
+ */
+class NewPaymentMethod implements PaymentMethodInterface {
+	/**
+	 * Contains the payment provider payment method ID.
```

Comment:
```suggestion
	 * External ID of the payment method.
```

I like the description of the constructor parameter better, let's use it here as well.

Reply:
Addressed https://github.com/Automattic/woocommerce-payments/pull/7020/commits/080ed7b9a384a64332319cdcd7a4d716d66d5871